lib/python/gladevcp/tooledit_widget.py

```diff
@@ -476,7 +476,7 @@ def col_editted(self, widget, path, new_text, col, filter):
         # validate input for float columns
         elif col in range(3,15):
             try:
-                self.model[path][col] = f"{float(new_text.replace(',', '.')):10.4f}"
+                self.model[path][col] =  "%10.4f" % locale.atof(new_text)
```

Comment:
```suggestion
                self.model[path][col] =  f"{locale.atof(new_text):10.4f}"
```

Please not re-introduce the old style formatting

Reply:
It's just about the line, not the whole commit. 
I thought it's easy to change that line...